### ISSUE TITLE
Bug fixes for building electron application

### ISSUE THREAD

**Issue by andrewjaykeller** [openexp/OpenEXP#57]
There will be several changes that will need to happen in order to actually get this built and distributable.
In `electron/main.js`

``` js
var app = require('app');

// browser-window creates a native window
var BrowserWindow = require('browser-window');
```

needs to become:

``` js
const electron = require('electron');
const app = electron.app; // module to control application life
const BrowserWindow = electron.BrowserWindow; // Module to create native browser window.

var mainWindow = null;
```

I site this issue as my reasoning and because when using electron-prebuilt version > 0.35.0 we need to change the way the main.js file requires electron thanks to a breaking change by the electron team.
https://github.com/maxogden/electron-packager/issues/205

**Comment by andrewjaykeller** [comment]
Also need to change the "main" file in the package.json file, to what, i'm not sure yet but seeing as it `server/start.js` does not exist, i'm thinking something else...

**Comment by andrewjaykeller** [comment]
in package.json change:
`"main": "server/start.js",` --> `"main": "electron/main.js",`

**Comment by andrewheusser** [comment]
@pushtheworldllc thanks!!  I'll make these changes to OpenEXP

**Comment by andrewjaykeller** [comment]
@andyh616 i can do a PR, there is a bunch of other stuff that needs to happen too

**Comment by andrewheusser** [comment]
ah, ok, that would be awesome @pushtheworldllc

**Comment by teonbrooks** [comment]
@pushtheworldllc, PR welcome :)

**Comment by andrewjaykeller** [comment]
Fixed with #59
